src/libraries/Microsoft.PowerFx.Core/Public/DependencyFinder.cs

```diff
@@ -31,6 +31,11 @@ public override void Visit(FirstNameNode node)
         {
             var info = _binding.GetInfo(node);
 
+            if (info == null)
+            {
+                return;
+            }
```

Comment:
Can you add some comment about when this can be null?

Reply:
Sure - Occurs when an identifier is used